### ISSUE TITLE
[Meeting] 01/19/2023

### ISSUE THREAD

**Issue by yehyunsuh** [yehyunsuh/Total-Knee-Replacement-Post-Surgical-Assessment#16]
## What
Have shown some progress after last meeting #15 
<img width="2672" alt="image" src="https://user-images.githubusercontent.com/73840274/213546137-d9a5bdd6-32d3-4c9a-81b0-0056ff5a4933.png">
However, the image seems to overfit to the dilated point
<br>

## Why
Need to find out the reason why
<br>

## How
- [x] Augmentation
- [x] weight the loss by class ( 1 ) maybe x100
- [ ] Progressive dilation
- [x] Overlay with original image for only validation set
- [x] Increase epoch to see the convergence and divergence
- [x] Log label accuracy to see whether the model is predicting the label correctly

**Comment by yehyunsuh** [comment]
Did not choose to use confusion matrix since the accuracy is not the important part in our project.

**Comment by yehyunsuh** [comment]
Increase epoch to see the convergence and divergence
![image](https://user-images.githubusercontent.com/73840274/213887740-5a952811-0890-499e-9d3a-a5245efa64f4.png)
- when the epoch increased, it diverged after 100 epochs even if the dilation was over 80 iterations. 
- it just guessed everything as 0

**Comment by yehyunsuh** [comment]
Close issue #18
